test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -1289,6 +1289,13 @@ public static Settings.Builder indexSettings(Version indexVersionCreated, int sh
             .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, replicas);
     }
 
+    /** Return consistent index settings for the provided shard- and replica-count. */
+    public static Settings.Builder indexSettings(int shards, int replicas) {
```

Comment:
You can call this inside the indexSettings() exactly above this one.

Reply:
Sort of :) That one is using the overload that sets the version so I'd need a third method to dry this up an then it's more LoC than just duplicating things like I did here :D